bin/rubydoop-runner

```diff
@@ -0,0 +1,9 @@
+# encoding: utf-8
+
+require 'rubydoop.jar'
+require 'rubydoop/job_runner'
+
+require 'puck'
+$: << "classpath:#{Puck::Jar::JAR_APP_HOME}/lib"
```

Comment:
As I've described in https://github.com/iconara/rubydoop/pull/32#issuecomment-122711939, I'm taking an alternative approach now. Hopefully I'll be able to use Puck as is.